ci-operator/jobs/openshift/kubernetes/openshift-kubernetes-master-presubmits.yaml

```diff
@@ -264,6 +264,79 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
I don't think we need a presubmit to always run. I think a periodic is sufficient then we can run the presubmit if needed

Reply:
Fixed